packages/shared/src/components/Feed.tsx

```diff
@@ -72,6 +73,10 @@ const SharePostModal = dynamic(
 const PostModal = dynamic(
```

Comment:
This is actually an `ArticlePostModal`, not blocker but a suggestion to change naming

Reply:
@el what do you think?

Reply:
Makes sense. changing it now